docs/index.md

```diff
@@ -1,5 +1,5 @@
 
-# Examples
+# Panel Chat Examples
```

Comment:
Renaming to Panel Chat Examples will keep the "Panel Chat Examples" when scrolling. This is much better than "Examples" if you want to copy-paste screen shots and share on social media.